src/main/java/co/inventorsoft/academy/service/RegistrationService.java

```diff
@@ -0,0 +1,79 @@
+package co.inventorsoft.academy.service;
+
+import co.inventorsoft.academy.model.Team;
+import co.inventorsoft.academy.dao.TeamDAO;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Scanner;
+
+import static co.inventorsoft.academy.enums.ColorText.RESET;
+import static co.inventorsoft.academy.enums.ColorText.RED;
+import static co.inventorsoft.academy.enums.ColorText.GREEN;
+import static co.inventorsoft.academy.enums.ColorText.BLUE;
+
+@Component
+@AllArgsConstructor
+public class RegistrationService {
+    /**
+     * list with team on this tournament
+     */
+    private TeamDAO teamDAO;
+
+    /**
+     * This method create teams and add pool of teams (LinkedList)
+     */
+    public void createTeam() {
+        Scanner in = new Scanner(System.in);
+        boolean exitFlag = true;
+        while (exitFlag) {
+            System.out.println(BLUE.getValue() + "Please choose number of option:\n" + " 1. Create team \n" + " 2. Start tournament\n" + " 3. Exit" + RESET.getValue());
+            String option = in.nextLine();
+            switch (option) {
+                /* Add new team to list  */
+                case "1":
+                    System.out.println("********************************************************************************************************************");
+                    System.out.print(GREEN.getValue() + "Input name of Team: " + RESET.getValue());
+                    String name = in.nextLine();
+                    System.out.print(GREEN.getValue() + "Input pilot #1 name: " + RESET.getValue());
+                    String pilot1 = in.nextLine();
+                    System.out.print(GREEN.getValue() + "Input pilot #2 name: " + RESET.getValue());
+                    String pilot2 = in.nextLine();
+                    teamDAO.addTeam(new Team(name, pilot1, pilot2));
+                    System.out.println("********************************************************************************************************************");
+                    break;
+                /* check team list and go to next step (start tournament) */
+                case "2":
+                    int size = teamDAO.size();
+                    if (isPowerOfTwo(size) && size >= 4) {
+                        exitFlag = false;
+                    } else {
+                        System.out.println("***********************************" + RED.getValue() + " You input " + teamDAO.size() + " teams please, input more teams! " + RESET.getValue() + "************************************");
```

Comment:
```suggestion
                        System.out.println("***********************************" + RED.getValue() + " You input " + size + " teams please, input more teams! " + RESET.getValue() + "************************************");
```
You have result fetched, no need to perform additional DB query.